cmake/ProcessOptions.cmake

```diff
@@ -384,20 +384,17 @@ endfunction()
 
 function( NEST_PROCESS_WITH_OPENMP )
   # Find OPENMP
-  if ( with-openmp )
-    if ( NOT "${with-openmp}" STREQUAL "ON" )
-      printInfo( "Set OpenMP argument: ${with-openmp}")
-      # set variables in this scope
-      set( OPENMP_FOUND ON )
-      set( OpenMP_C_FLAGS "${with-openmp}" )
-      set( OpenMP_CXX_FLAGS "${with-openmp}" )
-      set( OpenMP_CXX_LIBRARIES "${with-openmp}" )
-    else ()
-      find_package( OpenMP )
+  if ( NOT "${with-openmp}" STREQUAL "OFF" )
+    if ( NOT ${with-openmp} STREQUAL "ON" )
```

Comment:
Should `${with-openmp}` be with quotes `"${with-openmp}"`?

Reply:
Thanks for catching this, fixed.